.github/workflows/dev-build-deploy.yml

```
@@ -0,0 +1,62 @@
+      - name: Deploy to dev
+        uses: appleboy/ssh-action@v1.1.0
+        id: deploy-dev
+        if: contains(github.ref, 'develop')
+        with:
+          host: ${{ secrets.HOST_DEV }}
+          username: ${{ secrets.USERNAME }}
+          password: ${{ secrets.PASSWORD }}
+          port: ${{ secrets.PORT }}
+          script: |
+            sudo docker-compose pull ${{ secrets.DOCKERHUB_USERNAME }}/aics-api:latest
+            sudo docker-compose up -d
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)